lib/Runtime/Library/ArrayBuffer.h

```diff
@@ -142,15 +142,15 @@ namespace Js
             static FunctionInfo IsView;
             static FunctionInfo GetterByteLength;
             static FunctionInfo GetterSymbolSpecies;
-            static FunctionInfo Transfer;
+            static FunctionInfo Detach;
         };
 
         static Var NewInstance(RecyclableObject* function, CallInfo callInfo, ...);
         static Var EntrySlice(RecyclableObject* function, CallInfo callInfo, ...);
         static Var EntryIsView(RecyclableObject* function, CallInfo callInfo, ...);
         static Var EntryGetterByteLength(RecyclableObject* function, CallInfo callInfo, ...);
         static Var EntryGetterSymbolSpecies(RecyclableObject* function, CallInfo callInfo, ...);
-        static Var EntryTransfer(RecyclableObject* function, CallInfo callInfo, ...);
+        static Var EntryDetach(RecyclableObject* function, CallInfo callInfo, ...);
```

Comment:
nit: why not add this function in ENABLE_CONFIGOPTION only? same is true with EntryInfo::Detach

Reply:
I guess you're right. I was on the fence about it, but for no good reason.